web_app/telegram/notifications.py

```diff
@@ -0,0 +1,47 @@
+"""
+This module provides functionalities to send telegram notifications
+"""
+
+from decimal import Decimal
+from aiogram.exceptions import TelegramRetryAfter
+
+from web_app.db.crud import TelegramUserDBConnector
+from web_app.telegram import bot, logger
+
+from .texts import HEALTH_RATIO_WARNING_MESSAGE, NOTIFICATIONS_ENABLED_MESSAGE
+
+
+async def send_health_ratio_notification(
+    telegram_id: str, health_ratio: Decimal, retry_count: int = 1
+) -> None:
+    """
+    Send notification about health ratio to user
+    """
+    try:
+        telegram_db = TelegramUserDBConnector()
```

Comment:
move it out to constant, to not connect to db every time when this function is called